src/org/omegat/core/data/RealProject.java

```diff
@@ -699,6 +705,13 @@ public void compileProjectAndCommit(String sourcePattern, boolean doPostProcessi
                         null);
                 remoteRepositoryProvider.commitFiles(config.getTargetDir().getUnderRoot(),
                         "Project translation");
+                // Convert stats file name to relative
+                ProjectProperties.ProjectPath path = config.new ProjectPath(true);
+                path.setRelativeOrAbsolute(fn);
+                fn = path.getUnderRoot();
+                // remoteRepositoryProvider.copyFilesFromProjectToRepos(fn,null);
```

Comment:
To keep comaptibility with previous versions, we should keep commit text stat file.
When user community  recognize json stats file as standard team file, then we remove it from team repository.
 I think we may remove it in OmegaT 6.2.0.

We should notify the planning change in release 6.1.0.

Reply:
No problem for me, I commented out only because after requesting for it Manuel suddenly wanted to "save bandwidth"

Reply:
> No problem for me, I commented out only because after requesting for it Manuel suddenly wanted to "save bandwidth" and only the JSON was useful for him...

It was more a question than a request... In hindsight, since the stats file will always have a very small size, probably there's no need to worry about performance here.

Hence, while JSON is one always needed, I don't think it should be a problem to commit updates to stats files in several formats, if required or convenient.

Reply:
OK so I removed last patch then, now let @miurahr decide.